assets/elasticsearch/config.yaml

```
@@ -0,0 +1,30 @@
+related_processes:
+  - /usr/share/elasticsearch/jdk/bin/java
+  - /usr/share/elasticsearch/modules/x-pack-ml/platform/linux-x86_64/bin/controller
+queries:
+  - '''{"query": {"exists": {"field": "attr.tickets"}}, "size": 10000}'''
+  - '''{"query": {"term": {"id": 22419}}, "size": 10000}'''
+  - >
+    '{"query": {"bool": {"must": [{"wildcard": {"attr.message.msg": "log_release*"}}, {"match": {
+    "attr.message.session_name": "connection"}}]}}, "size": 10000}'
+  - >
+    '{"query": {"bool": {"must": [{"match": {"ctx": "initandlisten"}}], "should": [{"wildcard": {
+    "attr.message.msg": "log_remove*"}}, {"bool": {"must_not": [{"match_phrase": {"msg": 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)